src/components/experiment-tracking/run-metadata/index.js

```diff
@@ -105,9 +119,12 @@ const RunMetadata = ({
                   <td>
                     <p
                       className="details-metadata__notes"
+                      onClick={() => onTitleOrNoteClick(run.id)}
                       style={toggleNotes[i] ? { display: 'block' } : null}
                     >
-                      {sanitiseEmptyValue(run.notes)}
+                      {run.notes !== ''
+                        ? run.notes
+                        : '- Add notes here (Notes are searchable)'}
```

Comment:
similar to above, let's take out the `Notes are searchable` phrase until we have implemented the search feature?

Reply:
Removed.